tests/integration/cmor/_fixes/access/test_access_esm.py

```diff
@@ -0,0 +1,260 @@
+    extra_facets['frequency'] = frequency
+    extra_facets['exp'] = 'amip'
+    vardef = get_var_info(project='ACCESS', mip=mip, short_name=short_name)
+    cls = getattr(esmvalcore.cmor._fixes.access.access_esm, fix_name)
+    fix = cls(vardef, extra_facets=extra_facets, session={}, frequency='')
+    return fix
+
+
+def get_fix(mip, frequency, short_name):
+    """Load a variable fix from esmvalcore.cmor._fixes.cesm.cesm."""
+    fix_name = short_name[0].upper() + short_name[1:]
+    return _get_fix(mip, frequency, short_name, fix_name)
+
+
+def fix_metadata(cubes, mip, frequency, short_name):
+    """Fix metadata of cubes."""
+    fix = get_fix(mip, frequency, short_name)
+    cubes = fix.fix_metadata(cubes)
+    return cubes
+
+
+def check_tas_metadata(cubes):
+    """Check tas metadata."""
+    assert len(cubes) == 1
+    cube = cubes[0]
+    assert cube.var_name == 'tas'
+    assert cube.standard_name == 'air_temperature'
+    assert cube.long_name == 'Near-Surface Air Temperature'
+    assert cube.units == 'K'
+    assert 'positive' not in cube.attributes
+    return cube
+
+
+def check_pr_metadata(cubes):
+    """Check tas metadata."""
+    assert len(cubes) == 1
+    cube = cubes[0]
+    assert cube.var_name == 'pr'
+    assert cube.standard_name == 'precipitation_flux'
+    assert cube.long_name == 'Precipitation'
+    assert cube.units == 'kg m-2 s-1'
+    assert 'positive' not in cube.attributes
+    return cube
+
+
+def check_time(cube):
+    """Check time coordinate of cube."""
+    assert cube.coords('time', dim_coords=True)
+    time = cube.coord('time', dim_coords=True)
+    assert time.var_name == 'time'
+    assert time.standard_name == 'time'
+    assert time.bounds.shape == (1, 2)
+    assert time.attributes == {}
+
+
+def check_lat(cube):
+    """Check latitude coordinate of cube."""
+    assert cube.coords('latitude', dim_coords=True)
+    lat = cube.coord('latitude', dim_coords=True)
+    assert lat.var_name == 'lat'
+    assert lat.standard_name == 'latitude'
+    assert lat.units == 'degrees_north'
+    assert lat.attributes == {}
+
+
+def check_lon(cube):
+    """Check longitude coordinate of cube."""
+    assert cube.coords('longitude', dim_coords=True)
+    lon = cube.coord('longitude', dim_coords=True)
+    assert lon.var_name == 'lon'
+    assert lon.standard_name == 'longitude'
+    assert lon.units == 'degrees_east'
+    assert lon.attributes == {}
+
+
+def check_heightxm(cube, height_value):
+    """Check scalar heightxm coordinate of cube."""
+    assert cube.coords('height')
+    height = cube.coord('height')
+    assert height.var_name == 'height'
+    assert height.standard_name == 'height'
+    assert height.units == 'm'
+    assert height.attributes == {'positive': 'up'}
+    np.testing.assert_allclose(height.points, [height_value])
+    assert height.bounds is None
+
+
+def test_only_time(monkeypatch, cubes_2d):
+    """Test fix."""
+    var_list = ['tas', 'pr']
```

Comment:
I don't think it's necessary to test this for all variables. Just do it for `tas` (i.e., remove the loop and the `if`). Same applies to the other functions below.